### PR TITLE
Data description/clean data

### PR DESCRIPTION
**Pull request by lipcai**
1. upload the raw and clean data, 
2. update the proposal with data dercription,
3. add basic eda and data clean ipynb file

### CODE REVIEW

File: docs/proposal.md [hunk 1]
```diff
@@ -1,36 +1,41 @@
...
+
+Our role: Data Analysts at a consulting firm
+
+Target Audience: Movie enthusiasts 
+
+Millions of movies are published on a variety of channels; however, filtering excellent movies could be time-consuming. If we could provide a list of popular movies based on customers' preferences and illustrate approximation of box offices from different dimensions, they will save a large amount of time on searching for high-quality movies. To solve this challenge, we propose building a data visualization app that allows movie enthusiasts to explore the top 10 popular movies, box offices and other relevant information. Our app can be served as a great exploratory tool for people who like visualization and friendly-user-interface to extract various data at the same time with minimum efforts to select the filters.
+
+## Description of the Data
+
+This "movies" dataset is from the [Vega Datasets][1] collection. The dataset we are visualizing comes from the Vega dataset on Movies. The data was compiled by Vega to create a repository for example data to be used in their other packages as stated on their Vega Sources page.
```
Comment (zackt113) [inline docs/proposal.md]:
```suggestion
This "movies" dataset is from the [Vega Datasets][1] collection. The data was compiled by Vega to create a repository for example data to be used in other packages as stated on the Vega Sources page.

```

File: docs/proposal.md [hunk 2]
```diff
@@ -1,36 +1,41 @@
...
+
+This dataset contains 3,200 movies with 16 variables for each movie. Due to the time restriction of this project, we will only be visualizing 7 variables after 1980.
```
Comment (zackt113) [inline docs/proposal.md]:
```suggestion
This dataset contains 3,200 movies with 16 variables for each movie. By filtering out relevant features based on the purpose of our app, we decide to interactively visualize 7 variables of the movies that were created after 1980.

```

File: docs/proposal.md [hunk 3]
```diff
@@ -1,36 +1,41 @@
...
+
+Five of the variables we will be analysing describe the movie attributes including the title of the movie (Title), the duration of the movie, which genre the movie belongs to (Major_Genre), MPAA-rating and the year when the movie was originally released (Release_Date).
```
Comment (zackt113) [inline docs/proposal.md]:
```suggestion
There are five descriptive attributes of a movie including the title of a movie (Title), the duration of a movie (Running_Time_min), which genre the movie belongs to (Major_Genre), MPAA-rating and the year when the movie was originally released (Release_Date).

```

File: docs/proposal.md [hunk 4]
```diff
@@ -1,36 +1,41 @@
...
+
+The other two variables describe the outcomes of the movie through ratings (IMDB_Ratings) or US gross revenue. The dataset contains ratings from different sources, but we chose to visualize IMDB_Rating as it has 600 more values in the dataset than Rotten Tomatoes and relies on audience reviews. We will take the US gross revenue to summarize the revenue for each movie.
```
Comment (zackt113) [inline docs/proposal.md]:
```suggestion
The remaining two variables describe the popularity of a movie through ratings (IMDB_Ratings) and US gross revenue (US_Gross). The dataset contains ratings from two different sources, but we chose IMDB_Rating as it has fewer missing values than the Rotten Tomatoes rating. Additionally, we will use US Gross revenue as an approximation of the box office in North America.

```

File: src/movies_analysis_py.ipynb [hunk 5]
```diff
@@ -0,0 +1,401 @@
...
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.9.5"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 4
+}
```
Comment (zackt113) [inline src/movies_analysis_py.ipynb:237]: Could you please fix the error message in the ipynb file? Thanks..

**Review by zackt113 — CHANGES_REQUESTED** [review]

**Comment by lipcai** [comment]
Thanks

**Comment by adrianne-l** [comment]
Agree with Zack's comments on the proposal and once the error in ipynb is fixed. I think we are good on this part of data description and dataset. Good work! @lipcai

**Review by adrianne-l — COMMENTED** [review]
Agree with Zack's comments on the proposal and once the error in ipynb is fixed. I think we are good on this part of data description and dataset. Good work! @lipcai

File: src/movies_analysis_py.ipynb (hunk 5 above)
Comment (lipcai) [reply src/movies_analysis_py.ipynb:237]: have made the changes and need your approval on changes to merge  the PR

Comment (zackt113) [reply src/movies_analysis_py.ipynb:237]: Please check again for the ipynb file, it still shows KeyError: 'Major_Genre'. Thanks. And could you please solve the merge conflict below? Once you modify the ipynb file and solve the merge conflict, I will merge this PR! @lipcai

Comment (adrianne-l) [reply src/movies_analysis_py.ipynb:237]: I opened the file on jupyter notebook, it does not show the key error. However, it shows an indentation error for the docstring in Code Block 10 Line 64. Would you help to fix it? Thanks @lipcai 

`  Input In [10]
    Finds the number of movies of the most productive directors in the selected genre and
    ^
IndentationError: unexpected indent`

Comment (lipcai) [reply src/movies_analysis_py.ipynb:237]: I thought it is already comment out, so I would check that again. But I have no idea how to update the ipynb file when the PR is open, or should I just close this PR, and made all the uodates in a new pull request?

**Review by zackt113 — APPROVED** [review]

**Pull request merged**